Tasks/MavenV2/maventask.ts

```diff
@@ -286,6 +286,11 @@ async function execBuild() {
             });
 
             // Do not force an exit as publishing results is async and it won't have finished 
+        })
+        .fail(function (err) {
+            // Set task failure if get exception at step 5
+            console.error(err.message);
```

Comment:
Could you please describe it more precisely - at this step this error occured?

Reply:
Updated description, please take a look.